examples/hivemq-swarm/configmap_monitoring.yaml

```diff
@@ -0,0 +1,2372 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: swarm-hivemq-swarm-dashboard
```

Comment:
Does this name also have to be customized? 
If possible I would change this to `hivemq-swarm-grafana-dashboard`